spec/models/reports/appointment_scheduled_days_distribution_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+
+RSpec.describe Reports::AppointmentScheduledDaysDistribution, {type: :model, reporting_spec: true} do
```

Comment:
We should definitely use the helpers provided to `reporting_spec`s and freeze time for these specs. We should assert that boundary conditions take timezones into consideration correctly, etc. There's plenty of inspiration in our other reporting specs for this kind of stuff I believe.

Reply:
Our reporting view includes only the last 6 months of data for performance. The helper freezes time to `June 2021` so there will be no data to test with if we use the helper. We cannot freeze at a particular point in time because the spec will start failing after 6 months. 

If we understood correctly, the helper freezes time at `end of a month 23:00 IST ` and the month itself isn't important. What do you think about freezing at the `current month's end` in our specs?

Reply:
Ah I see. Yeah totally. The use of the helper is less important. Freezing time at the boundaries of months to make sure appointments are counted in the right months with respect to local timezones, that's more important. Yeah let's do it 👍

Reply:
Addressed in 3f3a794